adapters/handlers/rest/configure_api.go

```diff
@@ -650,28 +649,28 @@ func logger() *logrus.Logger {
 
 // sLogger returns an initialized standard logger
 // This logger should replace logrus in future
-func sLogger() *slog.Logger {
-	opts := slog.HandlerOptions{}
-	switch os.Getenv("LOG_LEVEL") {
-	case "debug":
-		opts.Level = slog.LevelDebug
-	case "warn":
-		opts.Level = slog.LevelWarn
-	case "error":
-		opts.Level = slog.LevelError
-	default:
-		opts.Level = slog.LevelInfo
-	}
-
-	var handler slog.Handler
-	if logTextFormat() {
-		handler = slog.NewTextHandler(os.Stderr, &opts)
-	} else {
-		handler = slog.NewJSONHandler(os.Stderr, &opts)
-	}
-
-	return slog.New(handler)
-}
+// func sLogger() *slog.Logger {
```

Comment:
thoughts(non-blocker): we should just delete dead code unless there is a strong reason to keep it around

Reply:
Agreed